### PR TITLE
Feature/show2 1200 coinbase wallet option style are broken

### PR DESCRIPTION
**Pull request by alantoa**
# Why

just fixed the Coinbase Wallet login option style.

# Before 
image

# After  

image

### CODE REVIEW

File: packages/app/providers/wallet-provider.tsx [hunk 1]
```diff
@@ -82,7 +82,11 @@ function WalletConnectQRCodeModalComponent(props: RenderQrcodeModalProps) {
           <PressableScale
             key={`wallet-cbw`}
             onPress={connectMobileSDK}
-            tw="my-2 flex-row items-center"
```
Comment (axeldelafosse) [inline packages/app/providers/wallet-provider.tsx:85]: Nice catch. We can't use Tailwind with animated styles anymore!

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/navigation/types.ts [hunk 2]
```diff
@@ -63,11 +63,35 @@ type BottomTabNavigatorParams = {
   profileTab: NavigatorScreenParams<ProfileStackParams>;
 };
 
-declare global {
-  // namespace ReactNavigation {
-  //   interface RootParamList extends BottomTabNavigatorParams {}
-  // }
-}
+type RootStackNavigatorParams = {
```
Comment (intergalacticspacehighway) [inline packages/app/navigation/types.ts:66]: 💯

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

**Pull request merged**